<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>banxingkongzhishumo</class>
 <widget class="QWidget" name="banxingkongzhishumo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>板形控制数模优化</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qtResource/resource.qrc">
    <normaloff>:/icons/icons/智能优化.png</normaloff>:/icons/icons/智能优化.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,20">
   <item>
    <widget class="NavigationBar" name="NavigationBar"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="tablePage"/>
     <widget class="QWidget" name="mechanismPage"/>
     <widget class="QWidget" name="feedbackControlPage">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch=""/>
     </widget>
     <widget class="QWidget" name="designPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavigationBar</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qtResource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
